src/components/structures/RightPanel.js

```diff
@@ -45,13 +45,9 @@ class HeaderButton extends React.Component {
         const TintableSvg = sdk.getComponent("elements.TintableSvg");
         const AccessibleButton = sdk.getComponent("elements.AccessibleButton");
 
-        return <AccessibleButton
-            className="mx_RightPanel_headerButton"
-            key={this.props.key}
-            onClick={ this.onClick }
-        >
+        return <AccessibleButton className="mx_RightPanel_headerButton" onClick={this.onClick} >
```

Comment:
/me thought should be `onClick={this.onClick}>` similar to the `<div` below not padding out its closing `>`